src/endpoints/ECatCourse/types/DateHandling.ts

```diff
@@ -0,0 +1,40 @@
+export enum DateHandlingType {
```

Comment:
Make sure to document this! If you wrote this file/these functions, slap your brand all over it!

Reply:
(not this specifically, more of the file as a whole!)

Reply:
Agreed :) some docs explaining what these are would be great, plus @author tags